cylc/flow/config_schema.py

```diff
@@ -163,17 +166,18 @@
             'task event handler retry delays': [VDR.V_STRING, ''],
             'tail command template': [VDR.V_STRING, ''],
             'batch systems': {
-                '__MANY__': {
```

Comment:
Why get rid of the `__many__` stuff?

I think there may possibly be a requirement (or at least desire) to, for example, submit jobs to a platform properly *or* run a quick script on the login node (e.g. housekeeping).

Reply:
I was thinking that we might have different platforms for the two use cases.

Reply:
You may have missed my other comments. Each platform should only have one batch system. If there are two batch systems, then you actually have two different platforms (that may happen to share some resources).

Reply:
See below discussion, background is a batch system.

Reply:
Please make no further comments on this thread. See the thread below: